Google.Api.Generator.Rest/Models/EnumModel.cs

```diff
@@ -42,15 +42,18 @@ public class EnumModel
         /// </summary>
         private string TypeName { get; }
 
-        public EnumModel(string name, JsonSchema schema)
+        public EnumModel(PackageModel package, Typ parentTyp, string name, JsonSchema schema)
         {
             IList<string> values = schema.Enum__;
             IList<string> description = schema.EnumDescriptions;
             Description = schema.Description;
             TypeName = name.ToMemberName() + "Enum";
+            // The exact value of the key doesn't particularly matter, so long as it's unique.
+            // This should be fine.
+            string enumStorageKey = $"{parentTyp.FullName}.{TypeName}";
```

Comment:
Just curious, are enums always nested types?

Reply:
Yup - they're nested within the request type that they're parameters for.